elegant/process_data.py

```diff
@@ -528,10 +528,12 @@ def measure(self, position_root, timepoint, annotations, before, after):
         # Remove the animal from the lawn if possible.
         center_tck, width_tck = annotations.get('pose', (None, None))
         if center_tck is not None:
-            lawn_mask &= worm_spline.lab_frame_mask(center_tck, width_tck, timepoint_image.shape).astype('bool')
+            worm_mask = worm_spline.lab_frame_mask(center_tck, width_tck, timepoint_image.shape).astype('bool')
+        else:
+            worm_mask = numpy.zeros_like(lawn_mask).astype('bool') # Doesn't mask out anything
```

Comment:
`worm_mask = 0` would work just fine here IIRC. I'll merge anyway and make that change myself.